cl/corpus_importer/import_columbia/columbia_utils.py

```diff
@@ -0,0 +1,841 @@
+
+FILED_TAGS = [
+    "filed",
+    "opinion filed",
+    "date",
+    "order filed",
+    "delivered and filed",
+    "letter filed",
+    "dated",
+    "release date",
+    "filing date",
+    "filed date",
+    "date submitted",
+    "as of",
+    "opinions filed",
+    "filed on",
+    "decision filed",
+    "date delivered",
+    "affirmed and opinion filed",
+    "dismissed and opinion filed",
+    "decided and entered",
+]
+DECIDED_TAGS = ["decided", "date decided", "decided on", "decided date"]
+ARGUED_TAGS = [
+    "argued",
+    "submitted",
+    "submitted on briefs",
+    "on briefs",
+    "heard",
+    "considered on briefs",
+    "argued and submitted",
+    "opinion",
+    "opinions delivered",
+    "opinion delivered",
+    "assigned on briefs",
+    "assigned on brief",
+    "opinion issued",
+    "delivered",
+    "rendered",
+    "considered on briefs on",
+    "opinion delivered and filed",
+    "orally argued",
+    "rendered on",
+    "oral argument",
+    "submitted on record and briefs",
+    "argued on",
+    "on reargument",
+]
+REARGUE_DENIED_TAGS = [
+    "reargument denied",
+    "rehearing denied",
+    "further rehearing denied",
+    "as modified on denial of rehearing",
+    "order denying rehearing",
+    "petition for rehearing filed",
+    "motion for rehearing filed",
+    "rehearing denied to bar commission",
+    "reconsideration denied",
+    "denied",
+    "review denied",
+    "motion for rehearing and/or transfer to supreme court denied",
+    "motion for reargument denied",
+    "petition and crosspetition for review denied",
+    "opinion modified and as modified rehearing denied",
+    "motion for rehearing andor transfer to supreme court denied",
+    "petition for rehearing denied",
+    "leave to appeal denied",
+    "rehearings denied",
+    "motion for rehearing denied",
+    "second rehearing denied",
+    "petition for review denied",
+    "appeal dismissed",
+    "rehearing en banc denied",
+    "rehearing and rehearing en banc denied",
+    "order denying petition for rehearing",
+    "all petitions for review denied",
+    "petition for allowance of appeal denied",
+    "opinion modified and rehearing denied",
+    "as amended on denial of rehearing",
+    "reh denied",
+    "opinion dissenting from denial of rehearing",
+]
+REARGUE_TAGS = ["reargued", "reheard", "upon rehearing", "on rehearing"]
+CERT_GRANTED_TAGS = [
+    "certiorari granted",
+    "petition and crosspetition for writ of certiorari granted",
+]
+CERT_DENIED_TAGS = [
+    "certiorari denied",
+    "certiorari quashed",
+    "certiorari denied by supreme court",
+    "petition for certiorari denied by supreme court",
+]
+UNKNOWN_TAGS = [
+    "petition for review allowed",
+    "affirmed",
+    "reversed and remanded",
+    "rehearing overruled",
+    "motion for rehearing overruled",
+    "review granted",
+    "decision released",
+    "transfer denied",
+    "released for publication",
```

Comment:
this code UNKNOWN_TAGS atleast is not used in this file and is used somewhere else.

Reply:
yes, it is not used, i forgot to remove it